crypto/pkcs11key/key_test.go

```diff
@@ -1,4 +1,4 @@
-// +build !nopkcs11
+// +build pkcs11
```

Comment:
Yes, but folks who are building on systems that don't have libtool-ltdl can build just the components that they need, which don't depend on `pkcs11key` anymore. E.g. `go build ./cli`